tests/integration/docker-compose.yml

```diff
@@ -56,6 +56,8 @@ services:
       OIDC_OP_LOGOUT_ENDPOINT_SECONDARY: "http://keycloak:8080/realms/secondary/protocol/openid-connect/logout"
       OIDC_OP_SET_ROLES_FROM_CLAIMS_SECONDARY: "true"
       OIDC_OP_ROLE_CLAIMS_PATH_SECONDARY: "realm_access.roles"
```

Comment:
I just noticed. Should this be named `OIDC_OP_ROLE_CLAIM_PATH_SECONDARY`?

Reply:
Good catch! Fixed in latest commit.